### DISCUSSION TITLE
Err: MATXEMP

### DISCUSSION THREAD

**Discussion by ghost** [Jyers/Marlin#1219] [Q&A] [answered]
A few days ago my printer was having issues printing and after taking off the silicon hotend cover I realized that melted plastic was building up underneath it. Upon cleaning the hotend, when I brushed over the part of the heater cartridge where it meets the wires with a pair of tweezers and my printer immediately started making a long beep with the following error message displayed:

![IMG_20210724_202238.jpg](https://user-images.githubusercontent.com/10644674/126903427-2b552ded-78b4-43e0-a932-cf1a5daa12ad.jpg)

This issue persists after being restarted and upon replacing not only the heater cartridge but the thermistor as well the error persists.

Does anybody know what the root cause of this issue is? It's entirely possible I messed up the thermistor or the heater cartridge while rewiring it.

**Comment by Cal-LF** [comment]
The thermistor mounting screw could be shorting the wires.

**Reply by ghost** [reply]
Thanks for the response.

In testing this idea I decided to remove the thermistor mounting screw entirely and let it hang loosely outside of the hotend. After turning the printer back on, the error persisted.

While the issue started the moment when I grazed the heater cartridge with my tweezers, I'm not entirely convinced that is the problem as there seems to be no damage to the original one and it's one of the components that doesn't send data back to the printer.

Building off this idea, am I right to assume that this issue has something to do with the thermistor? As I said in my initial post, I could have easily messed something up in installing it as I attempted to zip tie my new cables to my old cables and feed them through. Especially given that I did notice the thermistor bending backwards while being fed through and in many cases popping out of and getting caught on the cable organizing mesh.

**Comment by Cal-LF** [comment, marked as answer]
You have replaced the thermistor already, correct? Is there reason to think the new one is also bad? If the max-temp error is still occurring with a new thermistor, then the only thing left is the mainboard.
Thermistors are cheap. If you have another, install it to rule out another bad unit. 
It can be checked with an ohm meter. Connect to the leads and check for a reading. Warm the sensor and the resistance will change.

**Reply by ghost** [reply]
@Cal-LF, thanks for the help! I finally got around to replacing the board (or rather upgrading it to the 4.2.7 board) and it works properly again!

**Reply by Cal-LF** [reply]
@Dogs777 Glad you are up and printing!
